mrtt-ui/package.json

```diff
@@ -3,20 +3,28 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.9.0",
+    "@hookform/resolvers": "^2.8.8",
     "@mui/material": "^5.6.3",
     "@mui/styled-engine-sc": "^5.6.1",
+    "@mui/x-date-pickers": "^5.0.0-alpha.2",
```

Comment:
Is the alpha release the preferred option here? Ideally we should be sticking to stable releases.

Reply:
Yep I can get rid of sass and set a theme for MUI. With regards to date-pickers release, this is the latest ver
<img width="789" alt="Screen Shot 2022-05-02 at 9 06 55 AM" src="https://user-images.githubusercontent.com/26089140/166238927-7711c8f4-96dc-4cf1-9f4a-f6ef260d7737.png">
sion. The last three versions all have alpha in them... first one has 177k + downloads, maybe a safer bet?

Reply:
Sound like its the [direction they're taking](https://www.npmjs.com/package/@mui/x-date-pickers) with MUI. Let's just stick to the latest with the aim of switching to a stable release when possible.